MCprep_addon/materials/generate.py

```diff
@@ -29,6 +30,18 @@
 
 AnimatedTex = Dict[str, int]
 
+class PackFormat(Enum):
+	def _generate_next_value_(name, start, count, last_values):
+		return name
+	
+	SIMPLE = auto()
+	SEUS = auto()
+	SPECULAR = auto()
```

Comment:
Do not ever use `auto` for enums, use integers.

```suggestion
	SIMPLE = 1
	SEUS = 1
	SPECULAR = 2
```